client/web/src/enterprise/campaigns/preview/list/VisibleChangesetApplyPreviewNode.tsx

```diff
@@ -0,0 +1,267 @@
+                    </div>
+                </div>
+            </div>
+            <div className="d-flex justify-content-center">
+                <ApplyDiffStat spec={node} />
+            </div>
+            {/* The button for expanding the information used on xs devices. */}
+            <button
+                type="button"
+                aria-label={isExpanded ? 'Collapse section' : 'Expand section'}
+                onClick={toggleIsExpanded}
+                className="visible-changeset-apply-preview-node__show-details btn btn-outline-secondary d-block d-sm-none test-campaigns-expand-preview"
+            >
+                {isExpanded ? (
+                    <ChevronDownIcon className="icon-inline" aria-label="Close section" />
+                ) : (
+                    <ChevronRightIcon className="icon-inline" aria-label="Expand section" />
+                )}{' '}
+                {isExpanded ? 'Hide' : 'Show'} details
+            </button>
+            {isExpanded && (
+                <>
+                    <div />
+                    <div className="visible-changeset-apply-preview-node__expanded-section p-2">
+                        <ExpandedSection
+                            node={node}
+                            history={history}
+                            isLightTheme={isLightTheme}
+                            location={location}
+                            queryChangesetSpecFileDiffs={queryChangesetSpecFileDiffs}
+                            expandChangesetDescriptions={expandChangesetDescriptions}
+                        />
+                    </div>
+                </>
+            )}
+        </>
+    )
+}
+
+const ExpandedSection: React.FunctionComponent<
+    {
+        node: VisibleChangesetApplyPreviewFields
+        history: H.History
+        location: H.Location
+
+        /** Used for testing. **/
+        queryChangesetSpecFileDiffs?: typeof _queryChangesetSpecFileDiffs
+        /** Expand changeset descriptions, for testing only. **/
+        expandChangesetDescriptions?: boolean
+    } & ThemeProps
+> = ({ node, history, isLightTheme, location, queryChangesetSpecFileDiffs, expandChangesetDescriptions }) => {
+    if (node.targets.__typename === 'VisibleApplyPreviewTargetsDetach') {
+        return (
+            <div className="alert alert-info mb-0">
+                When run, the changeset <strong>{node.targets.changeset.title}</strong> in repo{' '}
+                <strong>{node.targets.changeset.repository.name}</strong> will be removed from this campaign.
+            </div>
+        )
+    }
+    if (node.targets.changesetSpec.description.__typename === 'ExistingChangesetReference') {
+        return (
+            <div className="alert alert-info mb-0">
+                When run, the changeset with ID {node.targets.changesetSpec.description.externalID} will be imported
+                from {node.targets.changesetSpec.description.baseRepository.name}.
```

Comment:
Added some **stronks** to match the previous case.

```suggestion
                When run, the changeset with ID <strong>{node.targets.changesetSpec.description.externalID}</strong> will be imported
                from <strong>{node.targets.changesetSpec.description.baseRepository.name}</strong>.
```